src/proxy/execute.go

```diff
@@ -260,3 +263,21 @@ func (spanner *Spanner) ExecuteOnThisBackend(backend string, query string) (*sql
 	defer txn.Finish()
 	return txn.ExecuteOnThisBackend(backend, query)
 }
+
+// If query is read and not in multi-statement txn, `load-balance` can take effect.
```

Comment:
// If query is read and not in multi-statement txn, `load-balance` can take effect.
// By using loadbalance hint can be forced to load balance.
->
// If the query is not in multi-statement txn, isExecOnRep  is called:
// 1. when the query is `Select`,  the config of the `loadBalance` can take effect.
// 2. By using loadbalance hint, directly decide load balance or not, not need to check `loadBalance`.

Reply:
> // If query is read and not in multi-statement txn, `load-balance` can take effect.
> // By using loadbalance hint can be forced to load balance.
> ->
> // If the query is not in multi-statement txn, isExecOnRep is called:
> // 1. when the query is `Select`, the config of the `loadBalance` can take effect.
> // 2. By using loadbalance hint, directly decide load balance or not, not need to check `loadBalance`.

done